tsl/src/compression/gorilla_impl.c

```diff
@@ -40,19 +40,19 @@ FUNCTION_NAME(gorilla_decompress_all, ELEMENT_TYPE)(CompressedGorillaData *goril
 	CheckCompressedData(n_total >= n_notnull);
 
 	/* Unpack the basic compressed data parts. */
-	Simple8bRleBitmap tag0s = simple8brle_bitmap_prefixsums(gorilla_data->tag0s);
-	Simple8bRleBitmap tag1s = simple8brle_bitmap_prefixsums(gorilla_data->tag1s);
+	const Simple8bRleBitmap tag0s = simple8brle_bitmap_prefixsums(gorilla_data->tag0s);
+	const Simple8bRleBitmap tag1s = simple8brle_bitmap_prefixsums(gorilla_data->tag1s);
```

Comment:
This is OK, but it is different from the other `const` usage: this makes `tag0s` and `tag1s` constant, the other cases made `*whatever` constant, but not `whatever`. Not sure if it was intentional or not, given that you haven't used the same principle in the other cases.

Reply:
Right, probably we should use `const void *data` in the `Simple8bRleBitmap`. I'll update it as well if it's possible.